shopify/mod.ts

```diff
@@ -18,6 +18,12 @@ export interface Props {
    */
   storefrontAccessToken: string;
 
+  /**
+   * @ttile Access Token
+   * @description Shopify admin access token.
+   */
+  adminAccessToken: string;
```

Comment:
Shouldn't you use Secret here?

Reply:
Yes, but Secrets are promises and today I can't use there, I will talk to candeia about it.

Reply:
I am using now